Formula/hledger.rb

```diff
@@ -43,3 +50,12 @@ def install
     system "#{bin}/hledger-ui", "--version"
   end
```

Comment:
This really needs better tests.

Reply:
I agree. It's hard to test a curses app but I'm working on it and hope to improve this with some future release.